src/main/index.ts

```diff
@@ -361,7 +361,8 @@ const store = new Conf<StoreSchema>({
       api_key: "2a69bcf769a7a28a8bf2f6a5100accad",
       secret: "46eea23770a459a49eb4d26cbf46b41c",
       token: null,
-      sessionKey: null
+      sessionKey: null,
+      scrobblePercent: 50
```

Comment:
@NovusTheory can probably give a better answer on this. But while testing this, was given an error for the `delete` when no value is set.

 I suspect that a Migration will need to be added for this value for it to apply the default value for people who are using YTMD.

Reply:
@Alipoodle You are correct that this needs a migration but it ideally would go under the next version that this would get merged into. Us maintainers can mark a note and just add it before the next version comes but this seems problematic long term.

Since I'm at least currently planning for 2.0.1 to correct all the bugs that have been introduced this probably can slip under that with a minor feature addition which happens to have bug fixes.

Reply:
We've discussed about what is the best solution in regards to doing migration and for now we're happy for user's to include it in the PR 😄 

We might change this up in the future if things don't make it into the next release etc.